packages/terra-list/docs/ListItem.md

```diff
@@ -0,0 +1,22 @@
+# List Item
+
+The list item provides a pattern for selection/selectability for li surrounded child content. The onSelect event is the primary means for interaction, as it yeilds the associated metaData when valid selection occurs, though individual onClick and onKeyDown events can be applied through the use of customProps if greated granularity is required.
+
+## Usage
+
+```jsx
+import React from 'react';
+import List, { Item } from 'terra-list/lib/List';
+
+<Item
+  key={itemData.key}
+  isSelectable
+  isSelected={false}
+  metaData={{ key: 'my-item-key' }}
+  onSelect={this.handleItemSelection}
+>
+  <p>blurb</p>
```

Comment:
We could replace the blurbs with "List item text" or something that doesn't remind me of the 1989 film staring Tom Hanks: https://www.imdb.com/title/tt0096734/

Reply:
Updated.